README.md

```diff
@@ -42,6 +42,8 @@ Certain configuration properties MUST be set via enviornment variables. The easi
 | DATA_TRUST_SERVICE | Yes | The base url for the data/trust service server. |
 | BASE_URL | Yes | The base url for this server. |
 | ADMIN_TOKEN | No | The value of the admin token to bypass authorization. If unset no admin token can be used. |
+| AUTH | No | If false, an access token will not be required to make requests. Defaults to true if unset. |
+| AUTH_REQUIRED_FOR_OUTGOING | No | If false, the backend app will not acquire an access token before sending requests to other servers. Defaults to true if unset. |
```

Comment:
Do we think it is worth making this a server by server thing? I don't think it would be hard but I'm not sure if that's overkill. I assume everyone at this connectathon will be doing no auth since that's what Dragon said but if someone was ready it would be fun to test it out

Reply:
If major EHRs are participating, I wonder if they even have the ability to turn off auth. We could do that but I think it's probably overkill since it will only be relevant for this sort of connectathon event, where the other participants don't have everything implemented yet.